pkgs/development/python-modules/tree-sitter-yaml/default.nix

```diff
@@ -0,0 +1,58 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildPythonPackage,
+  pythonOlder,
+  cargo,
+  rustPlatform,
+  rustc,
+  setuptools,
+  wheel,
+  tree-sitter,
+}:
+
+let
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "tree-sitter-grammars";
+    repo = "tree-sitter-yaml";
+    rev = "v${version}";
+    hash = "sha256-23/zcjnQUQt32N2EdQMzWM9srkXfQxlBvOo7FWH6rnw=";
+  };
+in
+buildPythonPackage {
+  pname = "tree-sitter-yaml";
+  inherit version src;
+  pyproject = true;
+  disabled = pythonOlder "3.9";
+
+  cargoDeps = rustPlatform.importCargoLock {
```

Comment:
In two places we have an importCargoLock, did you try fetchCargoVendor ?
If you did and it didnt work, no problems, we should just leave a comment as to what the error was and when potentially we might be able to change it.
If we have a possibility not to commit the cargo lock file to git that would be a big improvement.

Reply:
Reasonable suggestion, I have change to `fetchCargoVendor`.